LL-8791 LL-9146 (Market): release

LFBarreto opened the pull request:
## 🦒 Context (issues, jira)

[LL-8791]
[LL-9146]

- activate Market tab
- add analytics to market tab
- removes shared code in favor of LLC common logic for market provider
- escapes potential issues on ipcRenderer malformed logging

- removes dusk mode and updates theme color (to confirm with design if ok)

## 💻  Description / Demo (image or video)

<details>
    <summary> :camera_flash:  </summary>
    
![localhost_8080_webpack_index html_theme=dark](https://user-images.githubusercontent.com/11752937/150590569-a797b5a8-db58-41b2-bf5b-f59af6ccde9c.png)
![localhost_8080_webpack_index html_theme=light](https://user-images.githubusercontent.com/11752937/150590575-c591d569-0568-4c96-8d44-6e111702a06b.png)

</details>

<!-- please attached an image or even better a video that demo what this PR do -->

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: <!-- NB: Delivery incrementally with feature flagging is better than a very long PR. so prefer Option 1 if Option 2 takes more than a sprint -->
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [ ] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

<!--
If expectations aren't met, please document it carefully (on the reason you can't check it) and what do you need from maintainers.
-->


[LL-8791]: https://ledgerhq.atlassian.net/browse/LL-8791?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LL-9146]: https://ledgerhq.atlassian.net/browse/LL-9146?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

cgrellard-ledger committed:
LL-8811 currencies charts column is now hidden on the market page when the window size is too narrow

LFBarreto committed:
LL-8791 (Market): activate analytics + port LLC logic

LFBarreto committed:
(Market): update colors to new theme - dusk off

juan-cortes commented on src/preloader/logo.svg:
```diff
@@ -1,2 +1,24 @@
-
-<svg width="80" height="80" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient x1="50%" y1="100.845%" x2="50%" y2="2.104%" id="a"><stop stop-color="#165EDB" offset="0%"/><stop stop-color="#99B9FF" offset="100%"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><path d="M22.238 60.188c-7.548 4.451-11.341 6.657-11.379 6.617C-2.814 52.11-3.468 29.162 9.996 13.674a39.655 39.655 0 0 1  ...
+<svg width="80" height="80" xmlns="http://www.w3.org/2000/svg">
```
My svg eyes tell me this is the old icon, is that relevant, or should it be the new thing?
<img width="124" alt="image" src="https://user-images.githubusercontent.com/4631227/151428971-71d09f8e-6c2a-4df7-9bce-588bc93d5ab6.png">

juan-cortes commented on src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js:
```diff
@@ -49,8 +49,8 @@ const StepIndexContainer = styled.div`
-  background: rgba(100, 144, 241, 0.1);
-  color: #6490f1;
+  background: rgba(138, 128, 219, 0.1);
+  color: #8a80db;
```
What's up with all the color changes? 😕

juan-cortes replied:
Not that I care but the scope of the task is supposed to be activating the page + porting some logic

LFBarreto replied:
removes dusk mode and updates theme color `(to confirm with design if ok)`
we want to change the main color to match the new themes all other the app and not have discrepancy between market vs the rest.
the idea is ok with design expect the purple i put seems to be the wrong one :sweat_smile:

juan-cortes replied:
So then it's wrong Luiz, if the color is wrong, it's all wrong  (╯°□°)╯︵ ┻━┻

juan-cortes reviewed (commented):
I don't feel comfortable approving without understanding why there are so many UI changes 😓

cgrellard-ledger committed:
LL-8919 Market - removed filters footer (#4661)

* LL-8919 Market - removed filters footer and fixed some issues when filtering

* v2.37.0

* v2.37.1

* fix ci runs (#4672)

* test security fix (#4671)

Co-authored-by: Come Grellard <come.grellard@ledger.fr>
Co-authored-by: porenes-ledger <patrice.orenes-lerma@ledger.fr>
Co-authored-by: Gaëtan Renaudeau <renaudeau.gaetan@gmail.com>
Co-authored-by: Valentin D. Pinkman <valentin.d.pinkman@icloud.com>

imqdee committed:
LL-8813 - MARKET - Auto-fill swap source account (#4662)

* fix(market): auto-fill swap source account

Correctly set the correct source account when the user comes from the market page.

* fixup! fix(market): auto-fill swap source account

Co-authored-by: Alexandre Chabrolin <9203826+chabroA@users.noreply.github.com>

nabil-brn commented:
/generate-screenshots

ledgerlive committed:
update screenshots (linux)

ledgerlive committed:
update screenshots (windows-latest)

LFBarreto committed:
(Market): update purple colors + fixes

LFBarreto commented:
/generate-screenshots

ledgerlive committed:
update screenshots (linux)

ledgerlive committed:
update screenshots (windows-latest)

ledgerlive committed:
update screenshots (linux)

ledgerlive committed:
update screenshots (windows-latest)

LFBarreto committed:
fix issue in range missmatch

LFBarreto committed:
(Market): fix swap link in coin details page + breadcrumb

nabil-brn reviewed (approved):
Tested ✔️ 

---
cd43dc0b6 (HEAD -> LL-8791-market-analytics, ledger/LL-8791-market-analytics) (Market): fix swap link in coin details page + breadcrumb

Arnaud97234 committed:
fix flaky screenshot windows (#4697)

* fix flaky screenshot windows

* update screenshots (windows-latest)

* trigger ci

* changed test step for overflow setting

* update screenshots (windows-latest)

* waiting for tooltip to disappear

* Trigger Build

Co-authored-by: Team Live <team-live@ledger.fr>
Co-authored-by: Valentin D. Pinkman <valentin.d.pinkman@hey.com>
Co-authored-by: Gregor Gilchrist <gregor.gilchrist@ledger.fr>
